backend/tests/features/menu.feature

```diff
@@ -0,0 +1,272 @@
+      """
+      {
+        "name": "Batata",
+        "price": 10.00,
+        "description": "Não é a frita",
+        "image": "None",
+        "category": "Promoção",
+        "availability": "0 minutos"
+      }
+      """
+    Then o status code da resposta é "201"
+    And o corpo da resposta é:
+      """
+      {
+        "message": "Item Batata adicionado ao cardápio"
+      }
+      """
+    And a lista de itens no "cardápio" é:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+      | 0  | Batata | 10.00 | Não é a frita  | None  | Promoção | 0 minutos    |
+
+  Scenario: Atualizar um item do cardápio
+    Given que há um item registrado no "cardápio" com as informações:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+      | 0  | Batata | 10.00 | Não é a frita  | None  | Promoção | 0 minutos    |
+    And a informação requerida para atualizar é "name"
+    When o usuário faz uma requisição "PUT" para o endpoint "/menu/0" com as informações:
+      """
+      {
+        "price": 15.00,
+      }
+      """
+    Then o status code da resposta é "200"
+    And o corpo da resposta é:
+      """
+      {
+        "message": "Item Batata atualizado no cardápio"
+      }
+      """
+    And a lista de itens no "cardápio" é:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+      | 0  | Batata | 15.00 | Não é a frita  | None  | Promoção | 0 minutos    |
+
+  Scenario: Remover um item do cardápio
+    Given que há um item registrado no "cardápio" com as informações:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+      | 0  | Batata | 10.00 | Não é a frita  | None  | Promoção | 0 minutos    |
+    When o usuário faz uma requisição "DELETE" para o endpoint "/menu/0"
+    Then o status code da resposta é "200"
+    And o corpo da resposta é:
+      """
+      {
+        "message": "Item Batata removido do cardápio"
+      }
+      """
+    And a lista de itens no "cardápio" é:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+
+  Scenario: Adicionar um item sem nome ou preço ao cardápio
+    Given que não há itens registrados no cardápio
+    When o usuário faz uma requisição "POST" para o endpoint "/menu" com as informações:
+      """
+      {
+        "price": 10.00,
+        "description": "Não é a frita",
+        "image": "None",
+        "category": "Promoção",
+        "availability": "0 minutos"
+      }
+      """
+    Then o item não é adicionado ao cardápio
+    And o status code da resposta é "400"
+    And o corpo da resposta é:
+      """
+      {
+        "message": "O nome é requerido"
+      }
+      """
+    And a lista de itens no "cardápio" é:
+      | ID | Name   | Price | Description    | Image | Category | Availability |
+    When o usuário faz uma requisição "POST" para o endpoint "/menu" com as informações:
```

Comment:
Gherkin does not allow you to repeat the keywords 'given', 'when' and 'then' in the same scenario, please split this scenario into two.